wgengine/router/router_linux.go

```diff
@@ -164,10 +165,20 @@ func newUserspaceRouterAdvanced(logf logger.Logf, tunname string, linkMon *monit
 		ipt4: netfilter4,
 		ipt6: netfilter6,
 		cmd:  cmd,
-	}, nil
+	}
+
+	return r, nil
 }
 
 func (r *linuxRouter) Up() error {
+	if r.unregLinkMon == nil && r.linkMon != nil {
+		r.unregLinkMon = r.linkMon.RegisterRuleDeleteCallback(func(table uint8, priority uint32) {
+			if priority >= 5200 && priority < 5300 {
```

Comment:
not checking the `table`?

Reply:
Doesn't matter. Added a comment about why.